### ISSUE TITLE
Exception when binding float params in findBySql

### ISSUE THREAD

**Issue by shamyyl** [yiisoft/yii2-sphinx#8]
I am trying to use this code:
```php
GeoSubwaySphinx::findBySql(
	'SELECT id, name, geodist(:lat, :lng , latitude, longitude,{in=degrees, out=m}) as distance
	FROM geo_subway_plain_main
	WHERE distance <= 2000 and id_geo_parent = :id_geo_parent ORDER BY distance ASC',[
		':lat' => $lat,
		':lng' => $lng,
		':id_geo_parent' => $id_city,
	]
)->all();
```

but there is an exception:

Exception (Database Exception) 'yii\db\Exception' with message 'SQLSTATE[42000]: Syntax error or access violation: 1064 index geo_metro_plain_main: parse error: geodist() arguments can not be string
The SQL being executed was: SELECT id, name, geodist('55.788439', '49.119916' , latitude, longitude,{in=degrees, out=m}) as distance
            FROM geo_subway_plain_main
            WHERE distance <= 2000 and id_geo_parent = '2094' ORDER BY distance ASC' 

in /var/www/beboss.ru/shamil/www/vendor/yiisoft/yii2-sphinx/Schema.php:527

is it a bug or a feature?

**Comment by klimov-paul** [comment]
Relates https://github.com/yiisoft/yii2/issues/3513

**Comment by shamyyl** [comment]
thanks

**Comment by klimov-paul** [comment]
Resolved by commit 1e3bb8f0999b7b4b8dc868b125b9fed9a5e1322c

**Comment by klimov-paul** [comment]
Improved 9fc7835c6270a6dc5fc051d4f76d9c01e10c7d59
